t/page/house_wikidata.rb

```diff
@@ -19,8 +19,21 @@
     subject.title.must_equal 'EveryPolitician: Austria — Nationalrat'
   end
 
-  it 'should have popolo with wikidata' do
-    andrea = '0eedf2c9-01ea-44f4-bc6e-e5e4bf6d2add'
-    subject.popolo.persons.find_by(id: andrea).wikidata.must_equal 'Q493950'
+  it 'should pass a list of people with wikidata' do
+    res = subject.people_with_wikidata.select { |p| p[:name] == 'Cornelia Ecker' }
+    res.wont_be_nil
+  end
+
+  it 'should pass a list of people without wikidata as an array' do
+    subject.people_without_wikidata.must_be_empty
```

Comment:
It's a little worrying to have _only_ a test that this list is empty. This would still pass even if `people_without_wikidata` always returned `[]`. You probably want to add a test for another country too, where at least a few people don't have Wikidata mappings.